src/client/src/translations/whitelist_en.json

```diff
@@ -96,11 +96,7 @@
   "privacyModal.beingRedirected",
   "privacyModal.privacyStatement",
   "privacyModal.OK",
-  "projectName.projectNameTitle",
```

Comment:
seems like this is still being used too

Reply:
These are automatically generated when we build. @annielovesjs you should probably revert them to original if these changes were not intentionally made.

Reply:
Yup! Just reverted it